src/main/osd/osd_warnings.c

```diff
@@ -76,6 +76,20 @@ void renderOsdWarning(char *warningText, bool *blinking, uint8_t *displayAttr)
     static armingDisableFlags_e armingDisabledDisplayFlag = 0;
 
     warningText[0] = '\0';
+
+#ifdef USE_GPS_RESCUE
+    if (osdWarnGetState(OSD_WARNING_GPS_RESCUE_FAILING) &&
+      ARMING_FLAG(ARMED) &&
+      FLIGHT_MODE(GPS_RESCUE_MODE) &&
+      !gpsRescueIsOK()) {
```

Comment:
Please put logical operators at start of line - not sure about indentation in this case.
```suggestion
    if (osdWarnGetState(OSD_WARNING_GPS_RESCUE_FAILING)
      && ARMING_FLAG(ARMED)
      && FLIGHT_MODE(GPS_RESCUE_MODE)
      && !gpsRescueIsOK()) {
```

Reply:
Thanks - wasn't sure of how to do it, just copied the original approach.  Will put && first from now on.